forest/shared/src/cli/config.rs

```diff
@@ -157,6 +158,27 @@ impl Default for DaemonConfig {
     }
 }
 
+#[derive(serde::Deserialize, serde::Serialize, PartialEq, Eq, Clone)]
+pub struct TokioConfig {
+    pub worker_threads: usize,
+    pub max_blocking_threads: usize,
+    pub thread_keep_alive: Duration,
+    pub thread_stack_size: usize,
+    pub global_queue_interval: u32,
+}
+
+impl Default for TokioConfig {
+    fn default() -> Self {
+        Self {
+            worker_threads: num_cpus::get(),
+            max_blocking_threads: 512,
+            thread_keep_alive: Duration::from_secs(10),
+            thread_stack_size: 2 * 1024 * 1024,
+            global_queue_interval: 61,
```

Comment:
Could you point to the source of these defaults? Or did you choose them arbitrarily?

Reply:
These are default values of corresponding functions taken from https://docs.rs/tokio/0.2.1/tokio/runtime/struct.Builder.html . Should I point it out in code, @LesnyRumcajs ?

Reply:
Agree, or maybe we might really just want to expose the first four ones.

Reply:
If we make each of these params an `Option` then we should be good, we'd be redefining only if the user supplies his/her own arguments. Does it sound good to you?

Reply:
Sure.

Reply:
Fixed. @elmattic @LesnyRumcajs